proxy/core/src/main/scala/io/cloudstate/proxy/ProtobufGeneratedMessageRegisterFeature.scala

```diff
@@ -0,0 +1,32 @@
+package io.cloudstate.proxy
+
+import com.oracle.svm.core.annotate.AutomaticFeature
+import org.graalvm.nativeimage.hosted.Feature.QueryReachabilityAccess
+import org.graalvm.nativeimage.hosted.{Feature, RuntimeReflection}
+
+import scala.collection.JavaConverters._
+
+@AutomaticFeature
+final class ProtobufGeneratedMessageRegisterFeature extends Feature {
+
+  override def duringAnalysis(access: Feature.DuringAnalysisAccess) =
+    List(
+      "akka.protobuf.GeneratedMessage",
+      "akka.protobuf.GeneratedMessage$Builder",
+      "akka.protobuf.ProtocolMessageEnum",
+      "com.google.protobuf.GeneratedMessageV3",
+      "com.google.protobuf.GeneratedMessageV3$Builder",
+      "com.google.protobuf.ProtocolMessageEnum",
+      "scalapb.GeneratedMessage"
+    ).foreach { className =>
+      reachableSubtypes(access, access.findClassByName(className)).foreach { cls =>
+        RuntimeReflection.register(cls)
+        RuntimeReflection.register(cls.getMethods: _*)
```

Comment:
@dwijnand Shouldn't this be `getDeclaredMethods`?

Reply:
No `allPublicMethods` is `Class#getMethods`

Reply:
Yeah, but you removed this:

```
{
  "name":"com.google.protobuf.any.Any",
  "allDeclaredMethods":true,
  "allPublicMethods":true
},
```

Reply:
Hmm.  Ok, then I'll keep Any.